clippy_lints/src/strings.rs

```diff
@@ -169,3 +169,61 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for StringLitAsBytes {
         }
     }
 }
+
+/// **What it does:** Checks for usage of `.repeat(1)` since its equivalent to clone()
```

Comment:
Missing backticks around `.clone()`

Maybe add something like "while clone represents the intention better."

Reply:
Please also include https://github.com/rust-lang-nursery/rust-clippy/pull/3060#issuecomment-414235850 in the lint documentation.